acceptance/config/aio/options.rb

```diff
@@ -1,6 +1,7 @@
 {
   :type => 'aio',
   :is_puppetserver => true,
+  :puppetserver_confdir => '/etc/puppetlabs/puppetserver/conf.d',
```

Comment:
weird. i thought beaker converted the dashes in arguments and env vars to underscores for use in its options hash. but this is just an inconsistent convention...